src/main/java/at/hannibal2/skyhanni/api/GetFromSackAPI.kt

```diff
@@ -165,31 +165,41 @@ object GetFromSackAPI {
     )
 
     private fun commandValidator(args: List<String>): Pair<CommandResult, PrimitiveItemStack?> {
-        if (args.size <= 1) {
-            return CommandResult.WRONG_ARGUMENT to null
+        if (args.isEmpty()) return CommandResult.WRONG_ARGUMENT to null
+
+        val arguments = args.toMutableList()
+        val argsNull = args.last().last().toString().toIntOrNull() == null
+
+        if (argsNull) {
+            if (!config.defaultGFS) return CommandResult.WRONG_ARGUMENT to null
+
+            arguments.add(config.defaultAmountGFS.toString())
```

Comment:
Same thing, but without a mutable list
```suggestion
       val argsNull =  args.last().last().toString().toIntOrNull() == null
        val arguments = if(argsNull){
           if (!config.defaultGFS) return CommandResult.WRONG_ARGUMENT to null
           args + config.defaultAmountGFS.toString()
        } else {
          args
```